frontend/main-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Heading, Link } from "@digdir/design-system-react";
+import { ClipboardCheckmarkFillIcon } from "@navikt/aksel-icons";
+import classes from "./Header.module.css";
+
+export const Header = (): React.JSX.Element => {
+  return (
+    <header className={classes.header}>
+      <Link href="/" className={classes.headerLink}>
+        <Heading level={1} href="/">
```

Comment:
Maybe we can remove the ```href``` from the ```Heading``` component as it is not a Link? 😄

Reply:
Ah yes - nice find 😊